### ISSUE TITLE
vid2vid got bad results

### ISSUE THREAD

**Issue by ZhiyuXu0124** [NVlabs/imaginaire#32]
Hi, 

I am trying to use vid2vid to test my own data. The semantic segmentation map is from DeeplabV3+(checkpoint: [xception65_cityscapes_trainfine](https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/model_zoo.md)), and the results are as follows:

![](https://github.com/ZhiyuXu0124/AI_computervision/blob/master/vid2vid_result_mydata.png?raw=true)

I want to know whether the label of the vid2vid segmentation map is consistent with the label obtained by deeplabV3+? And can you give some suggestions on the results I got.

Many thanks! :)

**Comment by tcwang0509** [comment]
If you're using the pretrained cityscapes model, please use the cityscapes label policy: https://www.cityscapes-dataset.com/

**Comment by ZhiyuXu0124** [comment]
Hi, @tcwang0509 

My label is generated using deeplab, and it indeed complies with the cityscapes label policy.

**Comment by ZhiyuXu0124** [comment]
![](https://github.com/ZhiyuXu0124/AI_computervision/blob/master/deeplabV3+_result_mydata.png?raw=true)

**Comment by tcwang0509** [comment]
It seems the segmentation colors do not match?
Note that cityscapes has two sets of labels, one for training and one for evaluation. Be sure to select the training one.

**Comment by ZhiyuXu0124** [comment]
Thank you for your reminder, changing the training label to solve the problem.
